src/styles/theme/oc-button.scss

```diff
@@ -94,53 +61,79 @@
     }
   }
 
-  &.uk-button-primary {
+  &-primary {
     background-color: $action-primary;
     box-shadow: 0 4px 6px rgba($shadow-action-primary, 0.11), 0 1px 3px rgba($shadow-global, 0.08);
-    color: white;
+    color: $global-inverse-color;
 
     &:hover,
     &:focus {
       background-color: $action-primary-hover;
     }
   }
 
-  .uk-button-group & {
-    border-radius: 0;
-    // Prevent negative margin which is coming from UIkit
-    margin-left: 0 !important;
+  &-danger {
+    background-color: $danger-background;
+    color: $global-inverse-color;
 
-    &:first-of-type {
-      border-radius: 3px 0 0 3px;
+    &:hover,
+    &:focus {
+      background-color: $danger-background-hover;
     }
+  }
 
-    &:last-of-type {
-      border-radius: 0 3px 3px 0;
+  &:disabled {
+    border-color: lighten($global-muted-color, 25%);
+    color: lighten($global-muted-color, 25%);
+    cursor: default;
+
+    .oc-icon > svg {
+      fill: lighten($global-muted-color, 25%);
     }
+  }
 
-    &.uk-button-default {
-      border-left: none;
-      border-right: none;
+  &-group {
+    display: flex;
+    flex-flow: row wrap;
+   
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5393912)